views/moreInfo.ejs

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
here you are spitting out the whole JSON object, remember to pull out the individual parts like JSON.stringify(dataObj).Title or JSON.stringify(dataObj).Year